integreat_cms/cms/models/users/organization.py

```diff
@@ -26,10 +26,9 @@ class Organization(AbstractBaseModel):
     icon = models.ForeignKey(
         MediaFile,
         verbose_name=_("logo"),
-        on_delete=models.SET_NULL,
+        on_delete=models.PROTECT,
         related_name="icon_organizations",
-        blank=True,
-        null=True,
+        default="",
```

Comment:
```suggestion
```
An empty string does not make sense for a foreign key field. A foreign key can only contain the reference to the foreign object, or be `None`. Since we don't want to allow the latter, we have to make sure that all organization objects have a valid icon selected before we execute this migration on the production system.